backends/__init__.py

```diff
@@ -0,0 +1,71 @@
+import json
+
+__all__ = ['AbstractBackend', 'FileBackend']
+
+class AbstractBackend(object):
+
+    def __init__(self, *args, **kwargs):
+        super(AbstractBackend, self).__init__(*args, **kwargs)
+
+    def get_all_events(self):
```

Comment:
- Does RSVPBot need all past events? It would be easier if `get_all_events` could return something like all future events and the past two weeks worth of past events. That way, memory usage wouldn't grow linearly with time.